src/Avalonia.Controls/NativeControlHost.cs

```diff
@@ -118,7 +130,11 @@ private void UpdateHost()
             Debug.Assert(_currentRoot is not null);
 
             var bounds = Bounds;
-            var position = this.TranslatePoint(default, _currentRoot);
+            // Native window is not rendered by Avalonia
```

Comment:
Should probably use `new Rect(default, Bounds).Size).TransformToAABB(this.TransformToVisual(_currentRoot)` to get the rect.